cookbooks/vm/spec/integration/golang-dev/git_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+describe "vm::git", :"golang-dev" do
```

Comment:
wondering if we could use the `:"golang-dev"` tag in a way that would allow us to keep the specs more DRY (i.e. don't have the partly duplicated "golang-dev" and "full-node" specs side by side, but rather have only a single directory with all the specs, where some nested `describe` contexts would only be run if the "golang-dev" tag is passed.

Reply:
btw: as an alternative to using the tag, there is also the possibility to "skip" specific contexts when a given tag is not present. The subtle difference is that when the spec output is rendered it will actually display the skipped parts as well (in yellow rather than greed or red). might be useful in some situations

Reply:
> wondering if we could use the `:"golang-dev"` tag in a way that would allow us to keep the specs more DRY (i.e. don't have the partly duplicated "golang-dev" and "full-node" specs side by side, but rather have only a single directory with all the specs, where some nested `describe` contexts would only be run if the "golang-dev" tag is passed.

Good point. In larger projects I usually split the tests based on roles instead of stations. I would then tag the tests based on in which station which role is executed thereby avoiding duplicate tests.